sqlite/tool/mkkeywordhash.c

```diff
@@ -230,6 +230,9 @@ static Keyword aKeywordTable[] = {
   { "GLOB",             "TK_LIKE_KW",      ALWAYS                 },
   { "GROUP",            "TK_GROUP",        ALWAYS                 },
   { "GROUPS",           "TK_GROUPS",       WINDOWFUNC             },
+#if defined(SQLITE_BUILDING_FOR_COMDB2)
```

Comment:
Since this is an entirely new keyword (and not an "override" or an existing one), could you please move it to the comdb2 keywords section (below)?

Reply:
Done.